src/main/resources/db/migration/V202408091410__owner_table.sql

```diff
@@ -0,0 +1,9 @@
+create table owner(
```

Comment:
the name of the file should be the date it was created

Reply:
Ok,
Got it!